www/default.js

```diff
@@ -1,45 +1,50 @@
-'use strict';
+"use strict";
```

Comment:
Your changes in this file are not easily visible due to numerous formatting changes :(